src/Routes.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import MainContainer from 'src/MainContainer';
+import MovieComponent from 'src/MovieDetailed/MovieComponent';
```

Comment:
usually name of default import is the same with the folder

I expect `import MovieComponent from 'src/MovieComponent'`

not big issue

Reply:
and actually why do you need `Component` prefix? just `Movie` or `MovieDetails` :)

Reply:
Updated!